recipes/gembs-rs/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set version = "4.0" %}
+
+package:
+  name: gembs-rs
+  version: {{ version }}
+
+source:
+  url: https://github.com/heathsc/gemBS-rs/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: d2e4f844e4a381e5a9087c27ed67751ef2a3a2c2a97b02204ff6a483cad95cc9
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage("gembs-rs", max_pin="x") }}
+
+requirements:
+  build:
+    - make
+    - cmake >3.5
+    - autoconf
+    - automake
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - wget
```

Comment:
wget, zlib, htslib and rust should be removed or moved to `host`.
I see no usage of `wget` in build.sh